### PR TITLE
K8s: correct webhook name for 6.2.4-4

### PR DESCRIPTION
**Pull request by kaitlynmichael**
Jira: DOC-2066
Staged preview: 
https://docs.redis.com/staging/DOC-2066/kubernetes/deployment/openshift/openshift-cli/
https://docs.redis.com/staging/DOC-2066/kubernetes/deployment/quick-start/

### CODE REVIEW

**Review by nermiller — APPROVED** [review]

File: content/kubernetes/deployment/openshift/openshift-cli.md [hunk 1]
```diff
@@ -182,7 +182,7 @@ Each Redis Enterprise cluster requires at least 3 nodes. Single-node RECs are no
 1. Patch the validating webhook with the certificate.
 
     ```sh
-    kubectl patch ValidatingWebhookConfiguration redb-admission --patch "$(cat modified-webhook.yaml)"
+    kubectl patch ValidatingWebhookConfiguration redisenterprise-admission --patch "$(cat modified-webhook.yaml)"
```
Comment (yuvallevy2) [inline content/kubernetes/deployment/openshift/openshift-cli.md:185]:
```suggestion
    kubectl patch ValidatingWebhookConfiguration redis-enterprise-admission --patch "$(cat modified-webhook.yaml)"
```

File: content/kubernetes/deployment/openshift/openshift-cli.md [hunk 2]
```diff
@@ -215,9 +215,17 @@ If not limited, the webhook intercepts requests from all namespaces. If you have
 1. Apply the patch.
 
     ```bash
+    kubectl patch ValidatingWebhookConfiguration redisenterprise-admission --patch "$(cat modified-webhook.yaml)"
```
Comment (yuvallevy2) [inline content/kubernetes/deployment/openshift/openshift-cli.md:218]:
```suggestion
    kubectl patch ValidatingWebhookConfiguration redis-enterprise-admission --patch "$(cat modified-webhook.yaml)"
```

File: content/kubernetes/deployment/quick-start.md [hunk 3]
```diff
@@ -243,11 +243,18 @@ As part of the REC creation process, the operator stores the admission controlle
 
 1. Patch the webhook with the certificate.
 
+    ```sh
+    kubectl patch ValidatingWebhookConfiguration redisenterprise-admission --patch "$(cat modified-webhook.yaml)"
```
Comment (yuvallevy2) [inline content/kubernetes/deployment/quick-start.md:247]:
```suggestion
    kubectl patch ValidatingWebhookConfiguration redis-enterprise-admission --patch "$(cat modified-webhook.yaml)"
```

File: content/kubernetes/deployment/quick-start.md [hunk 4]
```diff
@@ -281,7 +288,7 @@ The operator bundle includes a webhook file. The webhook will intercept requests
 1. Apply the patch.
 
     ```sh
-    kubectl patch ValidatingWebhookConfiguration redb-admission --patch "$(cat modified-webhook.yaml)"
+    kubectl patch ValidatingWebhookConfiguration redisenterprise-admission --patch "$(cat modified-webhook.yaml)"
```
Comment (yuvallevy2) [inline content/kubernetes/deployment/quick-start.md:291]:
```suggestion
    kubectl patch ValidatingWebhookConfiguration redis-enterprise-admission --patch "$(cat modified-webhook.yaml)"
```

**Review by yuvallevy2 — COMMENTED** [review]
Please fix the name of the webhook

**Pull request merged**